src/components/templates/PartyMap/components/ScheduleItem/ScheduleItem.tsx

```diff
@@ -7,6 +7,8 @@ import { formatUtcSeconds } from "utils/time";
 import { useDispatch } from "hooks/useDispatch";
 import { retainAttendance } from "store/actions/Attendance";
 
+import "./ScheduleItem.scss";
```

Comment:
[FYI] This was the main fix for https://github.com/sparkletown/internal-sparkle-issues/issues/344